public/index.html

```diff
@@ -4,15 +4,18 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <title>Incorporation Application</title>
+    <title>BC Business Registry</title>
```

Comment:
The user experience should be the same between our apps (they should not know they're switching apps).